pandas/core/window/ewm.py

```diff
@@ -487,11 +497,74 @@ def mean(self, *args, engine=None, engine_kwargs=None, **kwargs):
 
             deltas = None if self.times is None else self._deltas
             window_func = partial(
-                window_aggregations.ewma,
+                window_aggregations.ewm,
+                com=self._com,
+                adjust=self.adjust,
+                ignore_na=self.ignore_na,
+                deltas=deltas,
+                normalize=True,
+            )
+            return self._apply(window_func)
+        else:
+            raise ValueError("engine must be either 'numba' or 'cython'")
+
+    @doc(
+        template_header,
+        create_section_header("Parameters"),
+        args_compat,
+        window_agg_numba_parameters,
+        kwargs_compat,
+        create_section_header("Returns"),
+        template_returns,
+        create_section_header("See Also"),
+        template_see_also,
+        create_section_header("Notes"),
+        numba_notes.replace("\n", "", 1),
+        window_method="ewm",
+        aggregation_description="(exponential weighted moment) sum",
+        agg_method="sum",
+    )
+    def sum(self, *args, engine=None, engine_kwargs=None, **kwargs):
+        if not self.adjust:
+            raise NotImplementedError("sum is not implemented with adjust=False")
+        if maybe_use_numba(engine):
+            if self.method == "single":
+                ewma_func = generate_numba_ewm_func(
+                    engine_kwargs,
+                    self._com,
+                    self.adjust,
+                    self.ignore_na,
+                    self._deltas,
+                    False,
+                )
+                numba_cache_key = (lambda x: x, "ewm_sum")
+            else:
+                ewma_func = generate_ewm_numba_table_func(
+                    engine_kwargs,
+                    self._com,
+                    self.adjust,
+                    self.ignore_na,
+                    self._deltas,
+                    False,
+                )
+                numba_cache_key = (lambda x: x, "ewm_sum_table")
+            return self._apply(
+                ewma_func,
+                numba_cache_key=numba_cache_key,
+            )
+        elif engine in ("cython", None):
+            if engine_kwargs is not None:
```

Comment:
is this hit in tests?

Reply:
Yup. This parameterization hits this: https://github.com/pandas-dev/pandas/pull/43871/files#diff-4fea2b6ab86ef8bb3c962ab4ee5667c2ea4df13bfab32ffcf03ec130451699dcR175